modules/programs/numbat.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib) mkIf;
+  cfg = config.programs.numbat;
+  tomlFormat = pkgs.formats.toml { };
+  configDir =
+    if pkgs.stdenv.hostPlatform.isDarwin then
+      "Library/Application Support/numbat"
+    else
```

Comment:
Just double checking, are you sure it doesn't respect xdg when its enabled on darwin?